lib/index.js

```diff
@@ -36,6 +36,10 @@ Plotly.register([
 
     require('./scattermapbox'),
 
+    require('./carpet'),
+    require('./scattercarpet'),
+    require('./contourcarpet'),
```

Comment:
OK. Let's leave carpet traces out of the cartesian bundle to start. We can always add then down the road in a future minor release if choose to.